content/overview/sidecar-files/sidecar.md

```diff
@@ -15,3 +15,7 @@ Your work is automatically synchronised to the sidecar files without the need to
 In addition to the sidecar files, darktable keeps all image-related data in its library database for fast access. An image can only be viewed and edited from within darktable if its data has first been loaded into the library database. This happens automatically when you first [import](../../module-reference/utility-modules/lighttable/import.md) an image. If an image is subsequently re-imported, the database will be updated from the contents of its XMP file.
 
 Once an image has been imported into darktable, the database entries take precedence over the XMP file. Subsequent changes to the XMP file by any other software are not visible to darktable -- such changes will be overwritten the next time darktable synchronizes the file. On request, darktable can be configured to search for updated XMP files at startup, offering a choice to update the database or overwrite the XMP file where changes are identified. This configuration can be changed in [preferences > storage > XMP sidecar files](../../preferences-settings/storage.md#xmp-sidecar-files).
+
+To import sidecar files created by other applications, see [importing sidecar files generated by other applications](sidecar-import.md).
+
+To load a sidecar file (e.g. one received from another darktable user), see [load sidecar file](../../module-reference/utility-modules/lighttable/history-stack/).
```

Comment:
Please reference the markdown file rather than the URL at which it will be rendered, so `/history-stack.md` rather than `/history-stack/`

Reply:
Done, thanks